### PR TITLE
[AGW] pipelineD: inout: remove vlan header for ingress packets

### LINKED ISSUE #2351: Strip vlan tag in ingress for UE traffic.

### PR DESCRIPTION
**Pull request by pshelar**
## Summary
In Multi APN setup, AGW support vlan network on SGi.
We need to remove this vlan header when transmitting packet over GTP. 

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Test Plan
`make test_python`

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/app/inout.py [hunk 1]
```diff
@@ -163,12 +162,22 @@ def _install_default_middle_flows(self, dp):
                                              resubmit_table=next_tbl)
 
         if self._mtr_service_enabled:
-            match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
+            match = MagmaMatch(vlan_vid=(0x0000, 0x1000),
```
Comment (xjtian) [inline lte/gateway/python/magma/pipelined/app/inout.py]: make this tuple a const

**Review by xjtian — APPROVED** [review]
Fix before merging

File: lte/gateway/python/magma/pipelined/app/inout.py (hunk 1 above)
Comment (pshelar) [reply lte/gateway/python/magma/pipelined/app/inout.py]: This is used in other pipelineD apps, I will take care of this as part of separate task.

**Review by koolzz — APPROVED** [review]
lgtm

**Pull request merged**